.github/renovate.json

```diff
@@ -4,16 +4,47 @@
   ],
   "baseBranches": [
     "main",
-    "/^release\\/v[0-9]+.x/",
-    "renovate-test"
+    "/^release\\/v[0-9]+.x/"
   ],
-  "prHourlyLimit": 2,
+  "prHourlyLimit": 12,
   "packageRules": [
     {
-      "matchBaseBranches": ["renovate-test"],
-      "matchDepNames": [
-        "kubernetes/kubernetes"
+      "matchDepNames": ["kubernetes/kubernetes"],
+      "separateMinorPatch": true,
+      "allowedVersions": ">=1.28.0"
+    },
+    {
+      "description": "Disable major and minor updates",
+      "matchDepNames": ["kubernetes/kubernetes"],
+      "matchUpdateTypes": ["major", "minor"],
+      "enabled": false
+    },
+    {
+      "groupName": "GitHub Workflow Actions",
+      "groupSlug": "gha-deps",
+      "matchManagers": [
+        "github-actions"
+      ]
+    },
+    {
+      "groupName": "Docker File Deps",
+      "groupSlug": "docker-bumps",
+      "matchManagers": [
+        "dockerfile"
       ]
     }
```

Comment:
We could potentially upstream this to `renovate-config`. Let's see how it goes in this repo first.